src/pages/Detail/AlertModal.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const IMAGE_LIST = [
+  { id: 1, img: 'http://placeimg.com/640/640/people' },
+  { id: 2, img: 'http://placeimg.com/640/640/tech' },
+  { id: 3, img: 'http://placeimg.com/640/640/animal' },
+  { id: 4, img: 'http://placeimg.com/640/640/nature' },
+];
+
+const AlertModal = ({ setIsClicked }) => {
+  const onCloseClick = () => {
+    setIsClicked(false);
+  };
+
+  return (
+    <AlertModalWrapper>
+      <IconBox>
+        <FontAwesomeIcon
+          icon="fa-solid fa-x"
+          size="lg"
+          onClick={onCloseClick}
+        />
+      </IconBox>
+      <AlertTitle>상품 특이 사항</AlertTitle>
+
+      <AlertContent>
+        <ItemArea>
+          <ItemImg
+            src="http://placeimg.com/640/640/people"
+            alt="제품 이미지"
+            width="80px"
+          />
+          <ItemInfo>
+            <ItemCode>DZ5485-612</ItemCode>
+            <ItemEnglishName>
+              Jordan 1 Retro High OG Chicago 2022
+            </ItemEnglishName>
+            <ItemKoreanName>조던 1 레트로 하이 OG 시카고 2022</ItemKoreanName>
+          </ItemInfo>
+        </ItemArea>
+        <ItemDesc>
+          <CommonNotice>
+            아래 개체별 특성을 확인하시고 거래 부탁드립니다. 개체별 특성에 따른
+            <HighlightText> 환불이나 교환은 불가능</HighlightText>
+            하므로 주의 부탁드립니다.
+          </CommonNotice>
+          <BoldText>01.</BoldText>
+          <CommonNotice>
+            해당 모델 가죽 소재 특성상 발생되는 이염 및 얼룩 다수 발견
+          </CommonNotice>
+          {IMAGE_LIST.map(({ id, img }) => {
+            return <ImgBox key={id} src={img} />;
+          })}
```

Comment:
how about
```suggestion
          {IMAGE_LIST.map(({ id, img }) => <ImgBox key={id} src={img} /> )}
```
함수 body문에 다른 선언 없이 바로 return  해주고 있으므로 위 처럼 바꾸면 가독성이 더 좋아질 것 같습니다!!